deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```
@@ -158,6 +158,12 @@ objects:
     - watch
     - update
     - patch
+  - apiGroups:
+    - authorization.k8s.io
+    resources:
+    - selfsubjectaccessreview
```

correct me if I'm wrong, but it should be in the plural form, right?:
```suggestion
    - selfsubjectaccessreviews
```

---

btw, is `selfsubjectaccessreview`  the only one that is used by UI, or does it also use SubjectAccessReview?

---

Yes, it should be in the plural form indeed! I keep missing it when reviewing those PRs :(

---

Updated!

The UI doesn't need to check access for users other than the current user, so SubjectAccessReview is not needed.